### DISCUSSION TITLE
NXP - Zephyr support for real -time (M33 and M7) cores of MX93 and S32Gx SoCs

### DISCUSSION THREAD

**Discussion by NXPDDFAE** [zephyrproject-rtos/zephyr#68251] [Q&A] [answered]
Hello,

What are NXP's plans to support Zephyr on the real-time cores of MX93 (M33) and S32G3 (M7) SoCs? 

I see Zephyr support for M4 and M7 real time cores of MX8mm and MX8M Plus EVKs.

Thanks,
NXPDDFAE

**Comment by DerekSnell** [comment, marked as answer]
Hi @NXPDDFAE ,
Today there is no Zephyr support for the real-time cores of MX93 (M33) and S32G3 (M7) SoCs.  And I am not aware of any public announcements from NXP for plans to add this support.  However, NXP is constantly working to enable Zephyr for many of our SoCs.  And the priorities are frequently determined by customer demand.  If there is interest in using Zephyr on SoCs not yet supported, please share the project details with your NXP sales or [distribution ](https://www.nxp.com/support/sample-and-buy/distributor-network:DISTRIBUTORS) contacts.

Best regards
